README.md

```diff
@@ -1 +1,51 @@
-# TODO
+# OSP-Qualification-Reports
+
+Qualification Reports recreated with every new OSP Release
+
+## How to create evaluation reports
+
+- Create a new branch from the `create-reports` branch (for instance, `my-reports`)
+  - Define the appropriate OSP environment and tools by updating `tools.csv`
+  - Define the models and projects by updating `models.csv`
+  - Go to the Github Action: [Automated Evaluation Reports and Projects](https://github.com/Open-Systems-Pharmacology/OSP-Qualification-Reports/actions/workflows/create-reports.yaml)
+    - Click on the button __Run workflow__ 
+      - Select your branch (for instance, `my-reports`)
+      - Click on the green button __Run workflow__ 
+  
+## What to do when reports are created
+
+When the evaluation reports are created, a pull request is triggered toward the `develop` branch.
+The pull request will allow users to review the updates in the reports and adopt the new version.
+
+## Models
+
+The `models.csv` file indicates which models should be run and qualified.
+The header includes the following fields:
+
+- __Execute__: If `TRUE` run the qualification. If `FALSE`, skip the qualification.
+- __Repository name__: Name of Github OSP repository from which to get the model, e.g. `7E3-Model` for [7E3-Model](https://github.com/Open-Systems-Pharmacology/7E3-Model)
+- __Released version__: Tag version of the model, e.g. `1.0`
+- __Snapshot name__: Name of the snapshot (`.json`) file, e.g. `7E3` for [7E3.json](https://github.com/Open-Systems-Pharmacology/7E3-Model/7E3.json)
+- __Workflow name__: Path of workflow R script that creates the function to run the qualification if not default.
+> [!TIP]
+> Leave blank cell if default path, `evaluation/workflow.R`, is used (this path is case insensitive). 
+- __Additional projects__: Url of additional project snapshots to export as `pksim5` projects, e.g. `Propofol-Pediatrics/refs/heads/main/Propofol-Pediatrics.json`
+> [!TIP]
+> If multiple projects are exported, they need to be separated by a pipe character: `|`, e.g. `X-Pediatrics.json|Y-Pediatrics.json`
```

Comment:
Let's replace the path in the example with a released version:
`Propofol-Pediatrics/refs/tags/v1.0/Propofol-Pediatrics.json`